themes/default/content/blog/pulumi-yaml/index.md

```diff
@@ -26,6 +26,8 @@ Pulumi YAML is a great option for smaller scale cloud infrastructure use cases.
 
 Like all other Pulumi languages, Pulumi YAML programs have access to all the core features of Pulumi’s infrastructure-as-code tooling, including native providers, secrets management, stack references, Pulumi Packages, and all the features of the Pulumi Service.  Critically, Pulumi YAML programs can interoperate seamlessly with the rest of the Pulumi ecosystem, consuming the outputs of other Pulumi programs and using Pulumi components built in existing Pulumi languages.  If the complexity of your Pulumi YAML projects increases, the new `pulumi convert` feature, enabled by the underlying [Pulumi CrossCode]({{< relref "/crosscode" >}}) technology, can be used to convert your Pulumi YAML program into your favorite general purpose Pulumi language.
 
+{{< youtube "QFCkubMhY3U" >}}
+
```

Comment:
Not entirely positive whether this is best so high up in the post - but will try it here for now.